api/scpca_portal/models/job.py

```diff
@@ -104,36 +111,125 @@ def get_sample_job(
             },
         )
 
+    @classmethod
+    def submit_created(cls, list_of_jobs: List[Self]) -> bool:
```

Comment:
In order to get around the issue with saving or not saving the job before submitting. Let's just query `CREATED` jobs instead. This should be easier to track. It can just return a list of jobs that were submitted.